nomos-blend/message/src/mock/error.rs

```diff
@@ -1,11 +1,13 @@
-#[derive(thiserror::Error, Debug)]
+#[derive(thiserror::Error, Debug, PartialEq)]
 pub enum Error {
     #[error("Invalid blend message format")]
     InvalidBlendMessage,
     #[error("Payload is too large")]
     PayloadTooLarge,
     #[error("Invalid number of layers")]
     InvalidNumberOfLayers,
+    #[error("Invalid public key")]
+    InvalidPublicKey,
```

Comment:
Instead of `InvalidNodeId`, I use this name because the trait `BlendMessage` uses the name `PublicKey` (not `NodeId`). In the `MockBlendMessage`, a node ID is used as a public key, but I think the error should use the generic term: public key.